pydantic_ai_slim/pydantic_ai/_griffe.py

```diff
@@ -125,3 +128,12 @@ def _infer_docstring_style(doc: str) -> DocstringStyle:
         'numpy',
     ),
 ]
+
+
+@contextmanager
+def _disable_griffe_logging():
+    # Hacky, but suggested here: https://github.com/mkdocstrings/griffe/issues/293#issuecomment-2167668117
+    old_level = logging.root.getEffectiveLevel()
+    logging.root.setLevel(logging.CRITICAL + 1)
```

Comment:
```suggestion
    logging.root.setLevel(logging.ERROR)
```
should solve the issue, and if there are any errors, we should probably not suppress them?